utils/utils.go

```diff
@@ -180,9 +196,9 @@ func ReadImageFromStaticPodDefinition(podFile, containerName string) (string, er
 }
 
 func HandleFilesWithCallback(folder string, action func(string) error) error {
-	return filepath.Walk(folder, func(path string, info os.FileInfo, err error) error {
+	return filepath.Walk(folder, func(path string, info os.FileInfo, err error) error { //nolint:wrapcheck
 		if err != nil {
-			return err
+			return fmt.Errorf("failed to walk: %w", err)
```

Comment:
lol

Reply:
lmao update to include the the faulty path in the log